express-get-json/server.js

```diff
@@ -0,0 +1,40 @@
+import express from 'express';
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const app = express();
+
+const array = [];
+
+app.use((req, res, next) => {
```

Comment:
Please make sure you're using the correct `app` method and parameters to only respond to `GET /api/grades` requests.